src/views/TechnicalAssistance/TrbAdminTeamHome.tsx

```diff
@@ -0,0 +1,648 @@
+      },
+      {
+        Header: t<string>('adminHome.requester'),
+        accessor: 'requesterInfo.commonName',
+        Cell: RequesterCell
+      },
+      {
+        Header: t<string>('adminHome.trbLead'),
+        accessor: 'trbLeadInfo.commonName',
+        Cell: TrbLeadCell
+      },
+      {
+        Header: t<string>('adminHome.status'),
+        accessor: ({ status }) => t(`table.requestStatus.${status}`)
+      },
+      {
+        Header: t<string>('table.header.trbConsultDate'),
+        accessor: 'consultMeetingTime',
+        Cell: ({
+          value,
+          row
+        }: CellProps<
+          TrbAdminTeamHomeRequest,
+          TrbAdminTeamHomeRequest['consultMeetingTime']
+        >) =>
+          value ? (
+            formatDateLocal(value, 'MM/dd/yyyy')
+          ) : (
+            <UswdsReactLink
+              to={`/trb/${row.original.id}/initial-request-form/schedule-consult`}
+            >
+              {t('adminTeamHome.actions.addDate')}
+            </UswdsReactLink>
+          )
+      }
+    ];
+  }, [t]);
+
+  const {
+    canNextPage,
+    canPreviousPage,
+    getTableBodyProps,
+    getTableProps,
+    gotoPage,
+    headerGroups,
+    nextPage,
+    page,
+    pageCount,
+    pageOptions,
+    prepareRow,
+    previousPage,
+    rows,
+    setGlobalFilter,
+    setPageSize,
+    state
+  } = useTable(
+    {
+      columns,
+      globalFilter: useMemo(() => globalFilterCellText, []),
+      data: useMemo(
+        () =>
+          // Switch on open vs closed existing requests
+          requests.filter((d: any) => d.state.toLowerCase() === activeTable),
+        [activeTable, requests]
+      ),
+      autoResetSortBy: false,
+      autoResetPage: false,
+      initialState: {
+        sortBy: useMemo(() => [{ id: 'form.submittedAt', desc: true }], []),
+        pageIndex: 0,
+        pageSize: 10
+      }
+    },
+    useFilters,
+    useGlobalFilter,
+    useSortBy,
+    usePagination
+  );
+
+  rows.map(row => prepareRow(row));
+
+  return (
+    <div>
+      <h3 className="margin-top-0 margin-bottom-1">
+        {t('adminTeamHome.existingRequests.heading')}
+      </h3>
+      <div className="margin-bottom-1 line-height-body-5">
+        {t('adminTeamHome.existingRequests.description')}
+      </div>
+
+      {/* Download existing requests csv */}
+      <div className="margin-bottom-4 line-height-body-5">
+        <CsvDownloadLink
+          data={getTrbRequestDataAsCsv(requests)}
+          filename="existing-trb-requests.csv"
+        >
+          {t('adminTeamHome.existingRequests.downloadCsv')}
+        </CsvDownloadLink>
+      </div>
+
+      {/* Open | Closed requests tabs */}
+      <nav aria-label={t('adminTeamHome.existingRequests.tabs.label')}>
+        <ul className="easi-request-repo__tab-list margin-bottom-4">
+          <li
+            className={classnames('easi-request-repo__tab', {
+              'easi-request-repo__tab--active': activeTable === 'open'
```

Comment:
Tab text should be smaller to match Figma - looks like `font-body-2xs` works.

Active tab text should be blue.